File: codeforces/round-697/a_odd_divisor.py
def squeeze(n):
    if n == 1:
        print("NO")
    else:
        if n % 2 == 0:
            squeeze(n/2)
        else:
            print("YES")

cases = int(input())
for i in range(0, cases):
    n = int(input())
    squeeze(n)

File: codeforces/round-697/a_odd_divisor.ml

let rec squeeze n =
  if n == 1 then
    print_endline "NO"
  else
    if n mod 2 == 0 then
      squeeze (n / 2)
    else
      print_endline "YES";;

let cases = read_int () in


for i = 1 to cases do
  let n = read_int () in
  squeeze n
done